src/main/java/Traffic/Reservation.java

```diff
@@ -71,7 +71,7 @@ public int reservSpace(int number_of_space, Time newTime)
 
     public boolean isFree(int number_of_space)
```

Comment:
Is it for parking space or entire parking? Can You please rename method to more descriptive?